frontend/apps/core-app/src/components/NavHeader/LargeNavHeader.component.tsx

```diff
@@ -38,16 +41,22 @@ export const LargeNavHeaderComponent: React.FC<
           </Link>
           <Link
             to={{
-              screen: routes.recipientsRoot.name,
-              params: { screen: routes.recipientsRegistration.name },
+              screen: routes.recipientsList.name,
+              params: {
+                screen: routes.recipientsRegistration.name,
+                params: {
+                  formId: forms[0]?.id,
+                  databaseId: forms[0]?.databaseId,
+                },
+              },
```

Comment:
```suggestion
              screen: routes.recipientsRegistration.name,
              params: {
                  formId: forms[0]?.id,
                  databaseId: forms[0]?.databaseId,
              },
```
this way the url would be `recipientsRegistration?formId={id}&databaseId={dbId}` instead of `recipientsList?screen=recipientsRegistration&params=%5Bobject%20Object%5D`, which does not lead to the correct screen

Reply:
Yeah I think this is a mistake from changing the nested navigators